source/cache-proxy/setup-squid-as-a-transparent-proxy.rst

```diff
@@ -11,9 +11,11 @@ Install the Package
 
 First, install the Squid package.
 
-#. Click **System > Packages**
-#. Scroll down until the "squid" package listing is visible
-#. Click |fa-plus| located to the right of the squid package description
+#. Click **System > Package Manager**
+#. Click Available Packages 
+#. Enter "squid" in the search bar and click search or scroll down until the "squid" package listing is visible
```

Comment:
You should make `squid` and inline block (double back ticks) for keyboard input (I'll add that to the style guide). I also think its better to bold key nouns, so **squid** package, for instance. (That was is more subtle, but I will put it in the style guide as well)

Reply:
squid is mentioned throughout this page, should all instances be bold? I went ahead and made the packages at the end of the page bold as well, squidguard and lightsquid, but am unsure if making every instance of squid bold is wanted.

Reply:
In this case "squid" should be bold since it was a GUI keyword/label that the user should be looking for, to make it stand out. The other instances don't need to be bold.